fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -2221,6 +2222,11 @@ private PlanFragment exchangeToMergeFragment(PlanFragment inputFragment, PlanTra
         PlanFragment fragment = new PlanFragment(context.nextFragmentId(), mergePlan, dataPartition);
         inputFragment.setDestination(mergePlan);
         context.addPlanFragment(fragment);
+        if (inputFragment.getSink() instanceof OlapTableSink) {
+            DataSink sink = inputFragment.getSink();
+            inputFragment.cleanSink();
+            fragment.setSink(sink);
+        }
```

Comment:
? why add this again?

Reply:
because it's changed at the same time as the pr, I remove it.